workers/go/throughputstress/activities.go

```diff
@@ -2,9 +2,17 @@ package throughputstress
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"math/rand"
 
+	"go.temporal.io/api/workflowservice/v1"
+
```

Comment:
Not a big fan of all the blank lines in the imports. I know server and SDK do it here and there for legacy reasons, but I would encourage Go authors to just have two blocks: stdlib and non-stdlib.

Reply:
I dunno why GoLand refuses to do things the goimports way. I keep telling it to.

Reply:
I think the goimports way is not to mess with existing import blocks. So it'll sort, and IDEs will auto-add to the bottom, but if you introduce a blank line, it'll retain it.

Reply:
I've never touched them manually ever, so it's something the IDE is doing :shrug: